src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/feign/CaseDataStoreFeignClient.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.ccd.document.am.controller.feign;
+
+import feign.Headers;
+import feign.Response;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import uk.gov.hmcts.reform.ccd.document.am.configuration.FeignClientConfiguration;
+
+import java.util.UUID;
+
+@FeignClient(name = "CaseDataStoreClient",
+             url = "${caseDataStoreUrl}",
```

Comment:
Could you please ensure this URL is added for other environments as well to ensure its working for PR and AAT.

Reply:
I will check at PR env.